tools/autograd/gen_annotated_fn_args.py

```diff
@@ -24,13 +24,15 @@
 from tools.codegen.context import with_native_function
 from tools.codegen.model import *
 import tools.codegen.api.python as python
-from .gen_python_functions import should_generate_py_binding, is_py_torch_function, is_py_nn_function, is_py_variable_method
+from .gen_python_functions import should_generate_py_binding, is_py_torch_function, \
+    is_py_nn_function, is_py_linalg_function, is_py_variable_method
 
 def gen_annotated(native_yaml_path: str, out: str, autograd_dir: str) -> None:
     native_functions = parse_native_yaml(native_yaml_path)
     mappings = (
         (is_py_torch_function, 'torch._C._VariableFunctions'),
         (is_py_nn_function, 'torch._C._nn'),
+        (is_py_linalg_function, 'torch._C._linalg'),
```

Comment:
I'm not sure what this does. Maybe @hameerabbasi can review this?

Is this just verifying that all ops in the torch.linalg namespace have overrides?

Reply:
This generates type annotation for the function parameters which are used by test_overrides.py [here](https://github.com/pytorch/pytorch/blob/49b59e347217e200e7d244ec9a2490ff7dea4345/test/test_overrides.py#L553) to know what type of input to create. By default it will create a single Tensor and since multi_dot takes a TensorList this was causing the test to fail.

Reply:
This is correct, thanks for adding this @heitorschueroff!